model/task/task.go

```diff
@@ -2884,23 +2884,29 @@ func addStatusColorSort(key string) bson.M {
 							},
 							"then": 1, // red
 						},
+						{
+							"case": bson.M{
+								"$in": []interface{}{"$" + key, []string{evergreen.TaskKnownIssue}},
+							},
+							"then": 2,
```

Comment:
can you remind me how the number weight maps to a color? what color does 2 map to?

Reply:
I found that the color mapping actually happens completely in spruce and the color comments actually describe spruce implementation.

Reply:
Yes, the number is just a priority